invenio_app_rdm/theme/assets/semantic-ui/less/invenio_app_rdm/theme/views/card.overrides

```diff
@@ -70,7 +70,12 @@
     display: flex;
     align-items: center;
     background: transparent;
-    overflow: hidden;
+
+    img {
+      object-fit: scale-down;
```

Comment:
was this PR updated? I don't see the newly added placeholder class rules here

Reply:
Oups ... I did forget ... Now it's updated. Thanks a lot.

Reply:
could you also remove or increase the custom height rules above in line 69 ? it is breaking the default card w/h ratio. In addition, please remove the blank display of description if there is no text, otherwise it is rendered with unused white spaces (this part not in CSS, but react code)
![Screenshot 2022-05-31 at 14 31 50](https://user-images.githubusercontent.com/38131488/171173864-d5a7738d-49b2-44b9-8227-bb4a1e389762.png)

![Screenshot 2022-05-31 at 14 30 18](https://user-images.githubusercontent.com/38131488/171173674-f7fa5a4f-612a-4b9c-b7f5-e3338548af3b.png)

Reply:
please modify the height to have a square card :)

Reply:
Ok, will do :+1: 
FYI: By doing so, the [PR in react-invenio-forms](https://github.com/inveniosoftware/react-invenio-forms/pull/137) is not needed anymore

Reply:
I think it is good enough :) thank you for changing this